### ISSUE TITLE
plot_isosurface does not respect arguments

### ISSUE THREAD

**Issue by satra** [maartenbreddels/ipyvolume#129]
these two keyword arguments: `wireframe=True, surface=True` are not used anywhere in the function.

**Comment by eszterpapp** [comment]
Hello! Is there any solution yet?

**Comment by maartenbreddels** [comment]
No, if you want to try to attach this (it is pure Python, no JS knowledge required) feel free to do so.
